src/factories/series/LineSeriesContainer.ts

```diff
@@ -0,0 +1,30 @@
+module n3Charts.Factory.Series {
+  'use strict';
+
+  export class LineSeriesContainer extends Utils.BaseFactory {
+
+    public svg: D3.Selection;
+
+    constructor() {
+      super();
+    }
+
+    create() {
+      this.createContainer(this.factoryMgr.get('container').vis);
```

Comment:
I thought, that the `Factory.Container` should create and provide at least one more g node for all the chart data.
Then we could make the `Factory.Series.Line` that creates another g node inside of the `g.data` to store all line series and so on